### ISSUE TITLE
Typo in Stream documentation example 

### ISSUE THREAD

**Issue by poqudrof** [RedisGears/RedisGears#801]
Hello, 

I could not find a branch for the documentation, so I put this here, please redirect me if this should be done somewhere else. 

In the file: https://oss.redis.com/redisgears/readers.html#streamreader

> Event Mode
> class GearsBuilder('StreamReader').run(prefix='*', batch=1, duration=0, onFailedPolicy='continue', onFailedRetryInterval=1, trimStream=True)

I suppose it should be register: 

> Event Mode
> class GearsBuilder('StreamReader').register(prefix='*', batch=1, duration=0, onFailedPolicy='continue', onFailedRetryInterval=1, trimStream=True)

I'm new to Gears and it got me confused :smiley: . 

Cheers, 

Jeremy.

**Comment by MeirShpilraien** [comment]
@poqudrof yes you are right, thanks for reporting, will make sure to fix the docs.
Side note, we do not have branch for documentation, we update master and then cherrypick the relevant fixes to version branches. All the docs are here: https://github.com/RedisGears/RedisGears/tree/master/docs
